packages/graphql/src/translate/projection/subquery/translate-cypher-directive-projection.ts

```diff
@@ -93,7 +92,7 @@ export function translateCypherDirectiveProjection({
         res.params = { ...res.params, ...p };
         subqueries.push(...nestedSubqueriesBeforeSort, ...nestedSubqueries);
         predicates.push(...nestedPredicates);
-    } else if (entity instanceof CompositeEntity) {
+    } else if (context.schemaModel.isCompositeEntity(entity)) {
```

Comment:
I'm thinking, how about holding this information in the entity class itself?

```
entity.isComposite()
```

To be clear, I think we need to rethink a bit the whole composite entity, but maybe it will be a bit cleaner using a method in the entity itself, so we don't need the whole schemaModel for this check

Reply:
I like this idea